examples/e-commerce/src/app.mobile.css

```diff
@@ -0,0 +1,261 @@
+@media (max-width: 899px) {
+  /* Filters overlay */
+
+  .container-filters {
+    background: #fff;
+    border-radius: 16px;
+    left: 0;
+    max-width: initial;
+    position: fixed;
```

Comment:
I'm not sure what's wrong on iphone X here, it might be the translateY that makes it endup there?
![2019-06-26 11 38 32](https://user-images.githubusercontent.com/2982512/60169462-081fb700-9807-11e9-89e4-826de8acedb6.gif)

Reply:
I prepared a fix that I'm gonna send in algolia/instantsearch.js#3866. We can then propagate it to this PR.